<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuMapa</class>
 <widget class="QWidget" name="MenuMapa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Mapa</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/descarga.png</normaloff>../icons/descarga.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QStackedWidget" name="stackedWidget_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stackedWidget_3Page1" native="true">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Crear</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Crear Mapa</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>No. Edificios:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_10">
             <property name="text">
              <string>Agregar Nuevo Enlace</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>Agregar Nuevo Edificio</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Mostrar</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Mostrar</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Buscar</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>Buscar</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Edificio:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit">
             <property name="placeholderText">
              <string>Código</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QTextBrowser" name="textBrowser_2"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_8">
          <attribute name="title">
           <string>Recorridos</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton_12">
             <property name="text">
              <string>Profundidad</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pushButton_11">
             <property name="text">
              <string>Anchura</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2" colspan="2">
            <widget class="QTextBrowser" name="textBrowser_7"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLineEdit" name="lineEdit_7">
             <property name="placeholderText">
              <string>Origen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_9">
          <attribute name="title">
           <string>Rutas</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0" colspan="2">
            <widget class="QTextBrowser" name="textBrowser_8"/>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit_8">
             <property name="placeholderText">
              <string>Origen</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_13">
             <property name="text">
              <string>Anchura</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="lineEdit_9">
             <property name="placeholderText">
              <string>Destino</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_14">
             <property name="text">
              <string>Profundiad</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stackedWidget_3Page2" native="true">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Mostrar Edificios</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser_9"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_mostarEdificios">
             <property name="text">
              <string>Mostrar</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_10">
          <attribute name="title">
           <string>Modificar Edificio</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="placeholderText">
              <string>Código:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="3">
            <widget class="QTextBrowser" name="textBrowser_5"/>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit_5">
             <property name="placeholderText">
              <string>Nombre:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>Modificar</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_11">
          <attribute name="title">
           <string>Modificar Camino</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QComboBox" name="comboBox_3"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>-&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="comboBox_4"/>
           </item>
           <item row="0" column="3" rowspan="3">
            <widget class="QTextBrowser" name="textBrowser_6"/>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QLineEdit" name="lineEdit_6">
             <property name="placeholderText">
              <string>Distancia</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QPushButton" name="pushButton_9">
             <property name="text">
              <string>Modificar</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Eliminar Edificio</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0" colspan="3">
            <widget class="QTextBrowser" name="textBrowser_3"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Edificio:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="placeholderText">
              <string>Código</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>Eliminar</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>Eliminar Camino</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0" colspan="3">
            <widget class="QTextBrowser" name="textBrowser_4"/>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="comboBox_2"/>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>Eliminar</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>                    -&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Regresar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_16">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
